.github/workflows/release.yml

```diff
@@ -44,9 +50,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Build ext package
-        run: python -m build ext/
-      - name: Publish ext to PyPI
+      - name: Build package
+        run: python -m build
+      - name: Publish to PyPI
```

Comment:
Diff looks like it does because the jobs switched place. Felt more natural to swap place due to the new dependency